opta/core/generator.py

```diff
@@ -83,5 +84,9 @@ def gen(
 # Generate a tags override file in every module, that adds opta tags to every resource.
 def gen_opta_resource_tags(layer: "Layer") -> None:
     if "aws" in layer.providers:
+        # Remove all generated tags files, forcing re-generation
+        for tag_fn in Path(tf_modules_path).glob(f"**/{TAGS_OVERRIDE_FILE}"):
+            tag_fn.unlink()
```

Comment:
Note that I tested this by adding a `breakpoint()` prior to removing the file, and after removing the file.  I manually ran this locally to make sure the files are removed correctly, which they are.